### PR TITLE
Added ability to pass quality via CP plugin, pep8 fixes

### PR DESCRIPTION
**Pull request by liiight**
Added ability to pass quality via CP plugin, pep8 fixes, handled connection exceptions

### CODE REVIEW

**Comment by gazpachoking** [comment]
@liiight So are these actually quality requirements that it's filling? Maybe we should put them in `quality_requirements` field, make it actual `QualtyRequirements` object rather than `Quality` and change `movie_queue` `add` mode to look at that field first.

File: flexget/plugins/input/couchpotato.py [hunk 1]
```diff
@@ -30,43 +30,85 @@ def on_task_input(self, task, config):
...
         for movie in json['movies']:
+            quality = ''
             if movie['status'] == 'active':
+                if config.get('include_data'):
+                    for profile in profile_json['list']:
+                        if profile['_id'] == movie['profile_id']:  # Matches movie profile with profile JSON
+                            # Creates a string of flexget qualities from CP's qualities list
+                            converted_list = ', '.join([cp_to_flexget[quality] for quality in profile['qualities']])
+                            try:
+                                quality = qualities.Quality (converted_list)  # Return the best components from the list
```
Comment (gazpachoking) [inline flexget/plugins/input/couchpotato.py]: Got an extra space in here before the parens.

**Comment by liiight** [comment]
@gazpachoking I guess your suggestion would make sense more than the current implementation, but I think if movie_queue is being refactored with regards to quality input, we can give it some more though and make it even more robust and generic than it currently is.

**Comment by liiight** [comment]
@gazpachoking you wanna merge this or should I start looking into your suggestion of passing quality requirement instead?

**Comment by gazpachoking** [comment]
@liiight Yeah, I guess that sounds fine. If we fix up movie_queue quality picking we can revisit this.

**Pull request merged**